.github/workflows/e2e_api_tests.yml

```
@@ -161,29 +162,39 @@ jobs:
+            - name: 🧪 Run e2e tests-${{ matrix.group }}
               id: e2e-test
               if: success() && (github.event_name != 'workflow_dispatch' || ( github.event_name == 'workflow_dispatch' && (github.event.inputs.testsuite == 'E2E' || github.event.inputs.testsuite == 'All')))
-              timeout-minutes: 40
+              timeout-minutes: 30
               working-directory: tests/pw
               run: |
-                  npm run test:e2e
+                  if [[ "${{ matrix.group }}" == "e2e_1" ]]; then
+                      export E2E_TEST_RESULT="./tests/pw/playwright-report/e2e/${{ matrix.group }}/summary-report/results.json"
+                      export E2E_COVERAGE="./tests/pw/playwright-report/e2e/${{ matrix.group }}/coverage-report/coverage.json"
+                      export GROUP="${{ matrix.group }}" 
```

yamllint (1.35.1)

[error] trailing spaces

(trailing-spaces)